ArduPlane/mode.h

```diff
@@ -526,6 +529,11 @@ class ModeStabilize : public Mode
 
     void run() override;
 
+#if AP_PLANE_SYSTEMID_ENABLED
+    // does this mode support fixed wing systemid?
+    bool supports_fw_systemid() const override { return true; }
```

Comment:
I'd recommend not having STABILIZE included, the way it does attitude targets would make systemID a mess. I recommend FBWA, CRUISE and LOITER as the initial set of modes we support fixed wing systemID in

Reply:
Not sure what you mean by this. I will have to dig into the code to see what stabilize does.

Reply:
Removing stabilize as one of the flight modes for system ID feature.  You were right in that stabilize did not give good data that agreed with the predictions.